plugins/rollbar_dashboards/app/helpers/rollbar_dashboards/dashboards_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module RollbarDashboards
+  module DashboardsHelper
+    DASHBOARD_HEIGHT = 22 # em
+
+    def placeholder_size(settings)
```

Comment:
this is a global helper, needs a more unique name

Reply:
Isn't it being namespaced?